### PR TITLE
Support 202 HTTP responses

### PR DESCRIPTION
**Pull request by MaybeNetwork**
## Feature Summary and Justification

Adds HTTP response status code 202 to the list of successful status codes. The endpoint [api/read_all_messages](https://www.reddit.com/dev/api#POST_api_read_all_messages) returns this response.

## References

* https://www.reddit.com/dev/api#POST_api_read_all_messages

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -3,6 +3,13 @@ Change Log
 
 prawcore follows `semantic versioning <http://semver.org/>`_.
 
+Unreleased
+------------------
```
Comment (bboe) [inline CHANGES.rst:7]: For RST this line should only be as long as the characters above it.

**Review by bboe — APPROVED** [review]
Minor change necessary if I end up merging this one first -- it might need to changed on the other PR if you want me to merge that one first.

File: CHANGES.rst (hunk 1 above)
Comment (MaybeNetwork) [reply CHANGES.rst:7]: Ok, I'll fix that

**Comment by bboe** [comment]
Thanks for the PR! 🎆

**Pull request merged**